### ISSUE TITLE
Thruk 3.0 Backends menu display issue

### ISSUE THREAD

**Issue by Reborns** [sni/Thruk#1225]
As we can see on images there is font issue for this buttons .
On first picture Windows machine and Chrome browser
On second picture Linux machine with Chrome browser 
![Capture](https://user-images.githubusercontent.com/39957910/204733848-80adc252-dcf0-4f63-9f20-318928ce2c92.PNG)
![photo_2022-11-30_11-24-26](https://user-images.githubusercontent.com/39957910/204733864-74ec71f9-1601-4144-adc6-53ecd4819514.jpg)

**Comment by sni** [comment]
The CSS is the same for both. Sometimes browser have a minimal font size configured. There is not much Thruk can do about it. How about choosing a shorter backend name or move the shared description part to the end so it doesn't matter if it overflows or not.

**Comment by Reborns** [comment]
Of course, I can fix it myself, it's just a bug report, because there is no such bug in older versions.

**Comment by sni** [comment]
i'd say version 2 is a bit smaller in many aspects. Could you use the chrome inspector to see
if there is any difference in font or font size between linux and windows?
